cjworkbench/middleware/i18n.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+
+
+class SetCurrentLocaleMiddleware(object):
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
+
+    def __call__(self, request):
+        # Code to be executed for each request before
+        # the view (and later middleware) are called.
+        locale = None
+
+        if request.user.is_staff:
```

Comment:
Nix this if-statement. Any user who knows the `?locale=` trick should see our in-progress translations. That way, developers and translators can translate while logged out.